packages/stream-tile/src/tile-document.ts

```diff
@@ -52,6 +53,16 @@ export interface TileMetadataArgs {
    * identical documents will generate unique StreamIDs and be able to be updated independently.
    */
   deterministic?: boolean
+
+  /**
+   * If true, all changes to the 'controllers' array are disallowed.  This guarantees that the
+   * Stream will always have the same controller. Especially useful for Streams controlled by
```

Comment:
I'm not sure if the part about did:nft is too much detail to go into here or not.  Could just end the comment after the second sentence if we think this is too specific for the doccomments

Reply:
I think an example is fine.